### DISCUSSION TITLE
Structure: Add Ability to Bulk Edit Structure Template Field

### DISCUSSION THREAD

**Discussion by jaredchesebro** [ExpressionEngine/ExpressionEngine#3471] [Feature Requests]
### Structure: Add Ability to Bulk Edit Structure Template Field

**Is your feature request related to a problem? Please describe.**
Occasionally, I want to create a new template for some of my Structure entries and apply it to several entries at once but currently I have to edit each entry one a time.

![structure-fields](https://github.com/ExpressionEngine/ExpressionEngine/assets/21958491/8b9f0167-512e-4514-b430-d5bb81fe3524)

**Describe the solution you'd like**
I think adding the Structure Template field as an option to Bulk Edit would solve this issue.

![bulk-edit](https://github.com/ExpressionEngine/ExpressionEngine/assets/21958491/7cae4895-f7d3-425d-8dac-248b56845a8f)

**Comment by melbazany** [comment]
Found this, but uncertain if it's up to par for EE Pro? http://buildwithstructure.com/documentation/how_can_i_batch_update_templates_for_existing_entries/

**Reply by jaredchesebro** [reply]
This might be a nice temporary solution but a native solution would be great since Structure is now included in EE core.

**Comment by intoeetive** [comment]
Aim to get this added with https://github.com/packettide/expressionengine-private/pull/48/commits/d260c21938211310310e80e7b0e0725c3dc2cdcf
